faqs/basics.md

```diff
@@ -0,0 +1,15 @@
+# FAQs
+
+### Why do we need OlympusDAO in the first place?
+
+Crypto is an emerging digital economy in the early stages of a multi-decade economic boom. Dollar-pegged stablecoins dominate crypto and the dollar is controlled by FED. By using these stablecoins, we have given up crypto’s monetary sovereignty and opened crypto up to external dependencies that introduce systemic risks. While that was a logical risk to take in the early days, as we march towards mainstream adoption we should be aiming to regain sovereignty in the form of crypto’s own cross-chain protocol-controlled currency system.
+
+### OHM is backed not pegged.
+
+Each OHM is backed by DAI, not pegged to it. This means OHM can never trade below 1 DAI because the treasury backs every OHM with at least 1 DAI. It could trade above 1 DAI because there is no cap imposed by the protocol.
```

Comment:
```suggestion
Each OHM is backed by 1 DAI, not pegged to it. Because the treasury backs every OHM with at least 1 DAI, when OHM trades below 1 DAI, the protocol would buy back and burn OHM, pushing its price back up to 1 DAI. However, it could always trade above 1 DAI because there is no upper limit imposed by the protocol. Think pegged == 1, while backed >= 1.
```

Reply:
Good suggestion. Added a mention that protocol would buy back and burn OHM when it trades below 1 DAI.

Reply:
The content is good. I would just split into different sentences. It appears pretty stretched with this many commas. It doesn't read natural as a single big sentence like this.

Reply:
```suggestion
Each OHM is backed by 1 DAI, not pegged to it. Because the treasury backs every OHM with at least 1 DAI, the protocol would buy back and burn OHM when it trades below 1 DAI. This has the effect of pushing OHM price back up to 1 DAI. OHM could always trade above 1 DAI because there is no upper limit imposed by the protocol. Think pegged == 1, while backed >= 1.
```

Reply:
Yeah that looks better. Do you commit the latest suggestions or do you want me to do it? 😅

Reply:
Thanks. I will commit it now.